### ISSUE TITLE
v1.10 RC2: Closed Tickets and Sort Issue 

### ISSUE THREAD

**Issue by rpbaquing** [osTicket/osTicket#2459]
Hi,

I upgraded my v1.10 RC1 to RC2.
But when I checked the Closed tickets, it does not give me the same result for both versions.
And also the sort button, does not give the same output.

Left (RC1) | Right (RC2)
![image](https://cloud.githubusercontent.com/assets/12552461/9432251/9107d176-4a52-11e5-9d31-5cc7e49fcd84.png)

And the total number of Closed tickets is gone..

**Comment by greezybacon** [comment]
<img width="1064" alt="screen shot 2015-08-27 at 5 04 20 pm" src="https://cloud.githubusercontent.com/assets/672074/9534388/addb98ae-4cdd-11e5-9519-3090cce34f3a.png">

It was moved to the bottom of the queue to save time on every request. For systems with lots of tickets, our tests showed as much as 100ms used to count closed tickets on each request.

**Comment by rpbaquing** [comment]
Thanks @greezybacon 

But I think there is something wrong with the sort function.
By default, it will show me the oldest closed tickets. Then when I tried to sort them our by the most recently closed tickets, it will not show me the latest instead it will start from somewhere  else, like the image I attached above. It started from Ticket #00428 (right - v RC2) but we must see Ticket #01710 (left - v RC1)

Then I tried  to go to the last page of the closed tickets and see the latest closed tickets, but when I tried to sort again on that page, it wont bring me to the last closed ticket.

![1](https://cloud.githubusercontent.com/assets/12552461/9539037/11360df8-4d7e-11e5-8dd1-4891c659d1d3.png)


I also notice something on the url that there are some duplication like the 'p='.
